src/hooks/dns/useDnsOffchainStatus.ts

```diff
@@ -54,28 +53,19 @@ export const useDnsOffchainStatus = ({ name, enabled = true }: UseDnsOffchainSta
     enabled,
   })
 
-  const {
-    data: addressRecord,
-    isLoading: isAddressRecordLoading,
-    isCachedData: isAddressRecordCachedData,
-  } = useAddressRecord({
-    name,
-    enabled: enabled && !!dnsOffchainData,
-  })
-
-  const isLoading = isDnsOffchainDataLoading || isAddressRecordLoading
-  const isCachedData = isDnsOffchainDataCachedData || isAddressRecordCachedData
+  const isLoading = isDnsOffchainDataLoading
+  const isCachedData = isDnsOffchainDataCachedData
 
   const data = useMemo(() => {
     if (isLoading || isError) return undefined
     return {
       resolver: getOffchainDnsResolverStatus({ chainId, dnsOffchainData }),
       address: checkDnsAddressMatch({
         address,
-        dnsAddress: addressRecord?.value as Address | undefined | null,
+        dnsAddress: dnsOffchainData?.extraData as Address | undefined | null,
```

Comment:
`dnsOffchainData.extraData` is what Cloudflare DNS returns from the DNS name. Not sure if that's correct, but `useAddressRecord` is always null

Reply:
if the address value is resolving, it means the dns resolution is working correctly. if useAddressRecord is resolving to null, there's an issue with resolution

Reply:
I checked dns-query requests, they all return a correct value. but the hook keeps returning null. it doesn't return null only if address is already imported. and this issue happens both on mainnet and sepolia.